client/src/components/EditProfile/EditProfileTab/EditProfileTab.tsx

```diff
@@ -84,9 +84,28 @@ export default function EditProfileTab(): JSX.Element {
       setPhone(values.phone);
       setAddress(values.address);
       setDescription(values.description);
+      handlePost();
     },
   });
 
+  const handlePost = () => {
+    const requestOptions = {
```

Comment:
Please create the API call within a function under the helpers folder (refer to existing API calls and follow the same format)